spec.html

```diff
@@ -28546,23 +28359,27 @@ <h1>Math.fround ( _x_ )</h1>
       <emu-clause id="sec-math.hypot">
         <h1>Math.hypot ( _value1_, _value2_, ..._values_ )</h1>
         <p>`Math.hypot` returns an implementation-approximated value representing the result of the square root of the sum of squares of its arguments.</p>
-        <ul>
-          <li>
-            If no arguments are passed, the result is *+0*.
-          </li>
-          <li>
-            If any argument is *+&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If any argument is *-&infin;*, the result is *+&infin;*.
-          </li>
-          <li>
-            If no argument is *+&infin;* or *-&infin;*, and any argument is *NaN*, the result is *NaN*.
-          </li>
-          <li>
-            If all arguments are either *+0* or *-0*, the result is *+0*.
-          </li>
-        </ul>
+        <emu-alg>
+          1. Let _v1_ be ? ToNumber(_value1_).
+          1. Let _v2_ be ? ToNumber(_value2_).
+          1. If _v1_ is *NaN* or _v2_ is *NaN*, return *NaN*.
+          1. If _v1_ is *+&infin;* or _v2_ is *+&infin;*, return *+&infin;*.
+          1. If _v1_ is *-&infin;* or _v2_ is *-&infin;*, return *-&infin;*.
+          1. Let _hasNaN_ be *false*.
+          1. Let _onlyZero_ be *true*.
+          1. If _v1_ is not 0 and _v2_ is not 0, set _onlyZero_ to *false*.
```

Comment:
I think we can skip all of this duplication by constructing a list like

```
1. Let _args_ be a List containing _value1_, _value2_, and the elements of _values_ in order.
```

and then just looping over `_args_` below instead of `_values_`.